receiver/mongodbatlasreceiver/receiver.go

```diff
@@ -99,6 +100,8 @@ func (s *receiver) poll(ctx context.Context, time timeconstraints) (pdata.Metric
 				resourceAttributes.CopyTo(resource.Attributes())
 				resource.Attributes().InsertString("host.name", process.Hostname)
 				resource.Attributes().InsertString("process.port", strconv.Itoa(process.Port))
+				resource.Attributes().InsertString("process.typename", process.TypeName)
```

Comment:
Is this usually being used as one word? If not, maybe not`process.type_name` instead?